lib/bolt/plugin/env_var.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'concurrent/delay'
```

Comment:
This is unused

Reply:
The perils of copy/paste. Updated.

Reply:
The real joy is it's also unused in the place it was copy/pasted from :)

Reply:
I'll go ahead and remove it in #1568 then 👍